### ISSUE TITLE
Meta: "Label" in the default issue template should be "tag"

### ISSUE THREAD

**Issue by john-light** [aragon/aragon-monthly#24]
In the contributor documentation we are calling content labels "tags" but the issue template calls it a "label". For consistency's sake, we should call these labels "tags" wherever they are mentioned.

**Comment by Smokyish** [comment]
@john-light Can you give a specific file that still has label instead of tag used? I know there were some inconsistency in the mentions of `tag`s before but can't find any right now. The only label mention that i see is for the `bounty` label in GH which is correct, as those are labels here.

**Comment by john-light** [comment]
@Smokyish good question, I saw "label" used in this issue (created by you, coincidentally) 

https://github.com/aragon/aragon-monthly/issues/15

And I thought that you were using the default issue template that users see when they open a new issue. But I just tried opening a new issue and it looks like the "Label" field was removed. So I think I was just confused by the use of the word "Label" in that specific issue.

Closing this issue since I don't see "Label" misused anywhere else (not to imply you misused it here if I am misinterpreting your use of the "Label" field).
